invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{ cookiecutter.package_name }}/config.py

```diff
@@ -25,4 +25,4 @@ def _(x):
 SECRET_KEY = "{{ cookiecutter.secret_key }}"
 
 # Theme
-THEME_SITENAME = _("{{ cookiecutter.site_name }}")
+THEME_SITENAME = "{{ cookiecutter.site_name }}"
```

Comment:
This is important for overlays so they can extract their THEME_SITENAME for translations automatically.